fdbserver/KeyValueStoreMemory.actor.cpp

```diff
@@ -546,13 +575,21 @@ class KeyValueStoreMemory : public IKeyValueStore, NonCopyable {
 				}
 
 				if (loggingDelay.isReady()) {
+					state bool isHealthy = wait(is_cluster_healthy());
 					TraceEvent("KVSMemRecoveryLogSnap", self->id)
+					    .detail("IsHealthy", isHealthy)
```

Comment:
the code base has odd tabs and spacing, you need to basically follow what's already there